projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -66,7 +66,7 @@ function register_block() {
 	if (
 		/** This filter is documented in class.jetpack-gutenberg.php */
 		! apply_filters( 'jetpack_subscriptions_newsletter_feature_enabled', false )
-		|| ! class_exists( '\Jetpack_Memberships' )
```

Comment:
`use Jetpack_Memberships` is in place. No need for `\`.

Reply:
Re-added in [a147fc0](https://github.com/Automattic/jetpack/pull/30807/commits/a147fc0a3c9f8662a52ad75c866b4271f540b4de)

Reply:
Thanks!